frontend/src/metabase/lib/entities.js

```diff
@@ -465,18 +465,22 @@ export function createEntity(def) {
   const getLoading = createSelector(
     [getRequestState],
     requestState => requestState.loading,
+    { devModeChecks: { inputStabilityCheck: "never" } },
```

Comment:
:x: These 4 warnings are correct and can be fixed by introducing a constant for the empty object returned in line 463.